bin/buildBlueprints.js

```diff
@@ -130,40 +131,52 @@ console.log('...Reading Blueprints', argv.blueprintsPath);
 console.log('...cwd', process.cwd());
 
 var config = makeConfig(argv);
+var isTest = argv.target === TARGETS.TEST;
 
 build(config, function(stats) {
   if (stats.errors && stats.errors.length > 0 && !argv.watch) {
     console.log(colors.red('ERROR IN BUILD. Aborting.'));
     process.exit(1);
   }
 
-  if (argv.runTest) {
+  if (isTest) {
     console.log(colors.magenta(
       '\n   ******************************' +
       '\n   *       RUNNING TESTS        *' +
       '\n   ******************************'
     ));
 
     m = new Mocha({ reporter: mochaNotifier.decorate('spec') });
-    stats.assets.forEach(function(asset) {
-      m.addFile('./.test/' + asset.name);
+    glob(path.join(TEST_DIR, '/**/*.compiledtest'), function (err, files) {
+      files.forEach(function(asset) {
+        m.addFile(asset);
+      });
+      m.run();
+
+      // we want to remove these from the require cache while we have path
+      // references to them to ensure they get tested on the next rebuild
+      m.files.forEach(function(filePath) {
+        delete require.cache[require.resolve(path.resolve(filePath))];
+      });
     });
-    m.run();
 
-    // we want to remove these from the require cache while we have path
-    // references to them to ensure they get tested on the next rebuild
-    m.files.forEach(function(filePath) {
-      delete require.cache[require.resolve(path.resolve(filePath))];
-    });
-  }
-});
+    // Hacky way to handle webpacks file output
+    function cleanup(err) {
+      if (err) {
+        console.error(err.stack);
+      }
+
+      try {
+        rimraf.sync(path.join(process.cwd(), TEST_DIR));
```

Comment:
it can fail (https://github.com/isaacs/rimraf/blob/2af08bbbd0a03549b278414309dc5d8097699443/rimraf.js#L312), but i'm not sure that we actually care if it does (want to fail the tests).